### PR TITLE
Integrated StoryBook to App and configured stories location

### PR DESCRIPTION
**Pull request by rahil051**
2. Added LoginForm and App Stories

### CODE REVIEW

**Comment by mashhoodr** [comment]
@rahil051 can we fix up the Codacy issues above.

File: client/src/components/PrivateRoute.js [hunk 1]
```diff
@@ -1,45 +1,79 @@
...
+} from 'react';
+import {
+	connect
+} from 'react-redux';
 import PropTypes from 'prop-types';
 import {
-    Route,
-    Redirect
-  } from 'react-router-dom'
+	Route,
```
Comment (mashhoodr) [inline client/src/components/PrivateRoute.js]: @rahil051 why did you change the tabs of the file?

File: client/src/components/PrivateRoute.js [hunk 2]
```diff
@@ -1,45 +1,79 @@
...
-    // const { appState, userState, events } = state;
-    return state;
+const mapStateToProps = state => {
+	// const { appState, userState, events } = state;
+	return state;
+}
+
+const map
+
+// TODO: Doesn't set default props for the component
```
Comment (mashhoodr) [inline client/src/components/PrivateRoute.js]: set these in side the class as a static prop. add a check for this so its respected in the whole codebase

File: client/src/components/PrivateRoute.js (hunk 1 above)
Comment (rahil051) [reply client/src/components/PrivateRoute.js]: @mashhoodr the editor reformatted the code.

Comment (mashhoodr) [reply client/src/components/PrivateRoute.js]: @rahil051 but you committed the code, please ensure this doesnt happen again.

**Comment by mashhoodr** [comment]
@oahmedqureshi @rahil051 I cant tell if this is ready to be merged?

**Comment by mashhoodr** [comment]
@oahmedqureshi @rahil051 please leave an update here
